presentation/src/main/java/me/androidbox/presentation/agenda/viewmodel/AgendaViewModel.kt

```diff
@@ -34,6 +75,7 @@ class AgendaViewModel @Inject constructor(
                         displayMonth = agendaScreenEvent.date.month.toString()
                     )
                 }
+                fetchAgendaItems(agendaScreenEvent.date)
```

Comment:
When selecting a new date from the calendar fetch from the DB for events created on this date.